packages/react/src/components/Storage/File-Uploader/Previewer/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/** File Previewer */
+
+import React, { useRef, useState } from 'react';
+import { Button, Flex, Card, View } from '../../../../primitives';
+import { Tracker } from '../Tracker';
+import { FilePreviewerProps } from '../types';
+import { UploadTask } from '@aws-amplify/storage';
+
+export function Previewer({
+  fileNames,
+  level,
+  files,
+  onClose,
+}: FilePreviewerProps): JSX.Element {
+  const [percentage, setPercentage] = useState<number[]>([]);
```

Comment:
`percentage` is an array of numbers?

Reply:
I might refactor this, but my initial thought was the previewer would keep track of all the files percentages in a useState. Then each one of those percentages would be passed into the Tracker component.

Reply:
I'll remove it for now